### PR TITLE
Docker improvements

### PR DESCRIPTION
**Pull request by lelia**
Some small improvements to the current Docker process:

- Include `sample.json` configuration file in the image so users can try Kombucha right away
- Create a symlink so the `kombucha` executable can be run from anywhere in the container
- Improve the `README` documentation for interacting with the Docker image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,18 +211,81 @@ Because the Docker container will not have direct access to your local filesyste
...
+
+To avoid this first-run error, simply add the `--record` option to your arguments:
+
+```bash
+$ kombucha --record
+
+wrote a reference to: /app/__Snapshots__/headersSnap.json
+wrote a reference to: /app/__Snapshots__/ipSnap.json
+wrote a reference to: /app/__Snapshots__/responseHeadersSnap.json
+wrote a reference to: /app/__Snapshots__/userAagentSnap.json
```
Comment (peter-tomaselli) [inline README.md]: I think there’s a typo here: `Aagent`

File: README.md [hunk 2]
```diff
@@ -211,18 +211,81 @@ Because the Docker container will not have direct access to your local filesyste
...
+
+finished recording
+```
+
+```json
+$ cat /app/__Snapshots__/headersSnap.json
```
Comment (peter-tomaselli) [inline README.md]: This line should probably be inside a `bash` fence, not a `json` fence??

File: Dockerfile [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
+# Add a default config
+COPY sample.json ./kombucha.json
+
 # Build the release
-RUN swift build -c release
\ No newline at end of file
+RUN swift build -c release
+
+# Create build symlink
+RUN ln -s /app/kombucha/.build/release/kombucha /usr/local/bin/kombucha
```
Comment (peter-tomaselli) [inline Dockerfile]: Awesome!

File: README.md (hunk 1 above)
Comment (lelia) [reply README.md]: Oh interesting! Turns out this is actually a typo in the `sample.json` file so I will fix to reflect that 👍

File: README.md (hunk 2 above)
Comment (lelia) [reply README.md]: Yeah good call, I forgot it's including a `cat` command :) Done!

**Pull request merged**